features/core-verify-checksums.feature

```diff
@@ -37,9 +37,9 @@ Feature: Validate checksums for WordPress install
     And I run `wp core download --version=4.3`
 
     When I try `wp core verify-checksums`
```

Comment:
`I try` should be changed to `I run`, given you're changing this to a success statement.